commands/NotifyRandomOpepenSet.ts

```diff
@@ -1,6 +1,7 @@
 import { BaseCommand } from '@adonisjs/core/build/standalone'
 import Env from '@ioc:Adonis/Core/Env'
 import pad from 'App/Helpers/pad'
+import SetSubmission from 'App/Models/SetSubmission'
```

Comment:
Top level app imports are discouraged in commands, as the App isn't loaded yet when the code is called.

Refactor to import within the `run` function.